all-category-books create

RomanIsaiev opened the pull request:
commit-1

create 'book-markup.js'
create 'category-books-title.js'
create 'all-category-books.js'
create 'book-markup.css'

do get (getBooksForCategory) from 'api,js'

import renderAllBooksInCategory in main.js

create gallery in 'index.html'

RomanIsaiev committed:
all-category-books create

commit-1

create 'book-markup.js'
create 'category-books-title.js'
create 'all-category-books.js'
create 'book-markup.css'

do get (getBooksForCategory) from 'api,js'

import renderAllBooksInCategory in main.js

create gallery in 'index.html'

idnol commented on src/css/book-markup.css:
```diff
@@ -0,0 +1,107 @@
+.book-card {
+  width: 335px;
+  height: 535px;
+}
```
ось тут ідея погана як його адаптувати в подальшому? треба щоб ширина точно враховувалась в залежності від ширини встого блоку

idnol reviewed (changes requested):
виправ будь ласка те що я написав

RomanIsaiev committed:
update markup and books title

RomanIsaiev committed:
update category books title

added css style 'span"
reworked 'category-books-title.js'
change index.html markup

idnol committed:
Merge branch 'main' into books-markup
